attribute/value_test.go

```diff
@@ -38,6 +38,12 @@ func TestValue(t *testing.T) {
 			wantType:  attribute.INT64,
 			wantValue: int64(42),
 		},
+		{
+			name:      "Key.Int64() correctly returns negative keys's internal int64 value",
+			value:     k.Int64(-42).Value,
+			wantType:  attribute.INT64,
+			wantValue: int64(0),
```

Comment:
This should be 

```suggestion
			wantValue: int64(-42),
```

negative floating number attributes are supported. See: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute